ghdocs/Contributing/NewComponent.md

```diff
@@ -0,0 +1,157 @@
+
+* Once you feel your component is ready to be promoted to become a **Fundamental component**, please request for a review with a Fabric core members. From outside Microsoft, you can do this through a [GitHub issue](https://github.com/OfficeDev/office-ui-fabric-react/issues).
+
+* Once the review is complete, you should be able to promote your component to the **Fundamental components** category. That is a big milestone.
+
+#### Experimental components
+Experimental components are any new components that do not currently exist within our [Adobe XD Toolkit](https://static2.sharepointonline.com/files/fabric/fabric-website/files/officeuifabric_v4.0.0.zip). By default, our `create-component` script will place the component folder and files in `packages/experiments`. New components stay in the experimental phase until a Fabric core team member approves the migrations from Experimental to Fundamental. Experimental components are developed in the experiments project located in the `<root of project>packages/experiments` folder. We also view any components that were created as a prototype or proof of concept.
+
+#### Fundamental components
+Fundamental components are the official React representation of our [Adobe XD Toolkit](https://static2.sharepointonline.com/files/fabric/fabric-website/files/officeuifabric_v4.0.0.zip) and receive higher priority in respect to stabilization, bug fixes, accessibility and general design resourcing. Components __cannot__ be immediately added as a Fundamental type component as it will need a period of stabilization and API review. All components should always start off as Experimental. A good portion of the Fundamental components are currently integrated within some of our major products such as OneDrive, SharePoint and Outlook. Fundamental components are developed in the office-ui-fabric-react project in the `<root of project>/packages/office-ui-fabric-react` folder.
+
+### Start coding your component
+
+#### Creating a component via the command line
+1. This is the recommended way of creating a new component.
+3. Open a command prompt in the root directory of your project directory.
+2. Run `npm run create-component -- --name ExcitingNewComponent`.
+
+#### Creating a component manually
+1. From the root of the project, navigate to `packages/experiments/src/components/`
+2. Create a new folder here with your component name
+  * Our components use the Pascal Case naming convention.
+  * In this example the component will be called `ExcitingNewComponent`.
+3. Create a `ExcitingNewComponent.types.ts` file that will contain an interface for your component props
+
+  * @todo: review
+  * Import React
+    * `import * as React from 'react'`;
+  * Import ExcitingNewComponent
+    * `import { ExcitingNewComponent } from './ExcitingNewComponent'`;
+    * Note: This class and file don't exist yet but they will during step 4 of this small tutorial.
+  * Create an empty interface `IExcitingNewComponent`
```

Comment:
Create an empty interface (which may contain the public imperative API surface, such as `focus()`.